references.bib

```diff
@@ -1102,3 +1102,18 @@ @inproceedings{DBLP:conf/sp/DanezisG09
   bibsource = {dblp computer science bibliography, https://dblp.org}
 }
 
+@misc{bowe18zexe,
```

Comment:
Better reference?
- https://eprint.iacr.org/2002/088
- https://electriccoin.co/blog/new-snark-curve/

Reply:
Hum... better references with regard to what? This is the ZEXE paper.
If you suggest alternative references to list when introducing the BLS12-377 curve, then yes, we can still add more context around BLS curves and the BLS12 family (and associated references). That being said, the ZEXE authors have instantiated BLS12-377 from the BLS12 family so we should be pointing to this paper when introducing this specific curve (The Zcash article you mention above refers to another BLS12 curve - called BLS12-381 which is not used in Zeth, so I don't think this is a required reference here)

Reply:
When possible, let's try to refer to conference papers. Zexe was accepted and presented at iEEE S&P this year, so there's a new dblp entry we can use here: https://dblp.uni-trier.de/rec/conf/sp/BoweC0MMW20.html?view=bibtex

Reply:
Updated the reference.  Thanks.